src/main/java/com/zufar/onlinestore/payment/service/PaymentEventCreator.java

```diff
@@ -0,0 +1,23 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.stripe.exception.SignatureVerificationException;
+import com.stripe.model.Event;
+import com.stripe.net.Webhook;
+import com.zufar.onlinestore.payment.config.StripeConfiguration;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class PaymentEventCreator {
```

Comment:
add java doc please
it helps to understand this class for people who have no idea about stripe

Reply:
Done